tools/spelling/spelling_dictionary.txt

```diff
@@ -233,6 +233,10 @@ LHS
 hls
 integrations
 jkl
+js
+lang
+hls
+jkl
```

Comment:
some of these are already in the dictionary.

Reply:
changed

Reply:
"js" seems to be on line 826 https://github.com/envoyproxy/envoy/blob/main/tools/spelling/spelling_dictionary.txt#L826
"hls" seems to be on line 233
https://github.com/envoyproxy/envoy/blob/main/tools/spelling/spelling_dictionary.txt#L233

Reply:
Ok think I get this right now.